client/src/components/home/Navbar.js

```diff
@@ -19,16 +37,16 @@ function Navbar() {
               <Link to="/products/women">WOMEN</Link>
```

Comment:
I have some thoughts on improvements here but I think maybe I should see the UI you're working first